src/tests/ftest/util/server_utils_params.py

```diff
@@ -480,7 +480,7 @@ def __init__(self, base_namespace, index, provider=None, max_storage_tiers=MAX_S
         #   log_file:               map to D_LOG_FILE env
         #   env_vars:               influences DAOS I/O Engine behavior
         self.targets = BasicParameter(None, 8)
-        self.first_core = BasicParameter(None, 0)
+        self.first_core = BasicParameter(None)
```

Comment:
This means all tests will no longer have `first_core` in the config. Running `pr` + `Features: control ObjectMetadata` is probably good coverage of this, but are there any areas we should be concerned with?

Reply:
I'm going to change this to allow setting first_core: 0